php/Terminus/Models/Site.php

```diff
@@ -332,6 +333,19 @@ public function fetch(array $options = array()) {
   }
 
   /**
+   * Re-fetches site attributes from the API
+   *
+   * @return void
+   */
+  public function fetchAttributes() {
+    $response = $this->request->simpleRequest(
+      sprintf('sites/%s/settings', $this->get('id'))
+    );
+    $this->attributes = $response['data'];
+    $this->collection->sites_cache->update((array)$response['data']);
```

Comment:
We're storing the updated PHP version information. Since we're still using the cache, we need to update it once we change something recorded in the cache.